components/shared/BreadcrumbsFilter.vue

```diff
@@ -82,6 +84,10 @@ const collections = computed<Collection[]>(() =>
   )
 )
 
+const isCollectionSearchMode = computed(() => {
+  return useCollectionSearch().isCollectionSearchMode
+})
+
```

Comment:
don't need another computed ref no?

```suggestion
const { isCollectionSearchMode } = useCollectionSearch()
```